File: etc(+simulation)/swea_2805.py
T = int(input())

for test_case in range(1, T+1):
    n = int(input())
    board = [list(map(int, input().strip())) for _ in range(n)]

    result = 0
    mid = n // 2
    for i in range(n):
        row = board[i]
        valid = row[abs(i - mid): n - abs(i - mid)]
        result += sum(valid)
    
    print(f"#{test_case} {result}")

File: etc(+simulation)/swea_2805.java
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class Solution {
	public static void main(String[] args) throws Exception {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		StringBuilder sb = new StringBuilder();
		
		int T = Integer.parseInt(br.readLine());
		for(int testCase = 1; testCase <= T; testCase++) {
			int N = Integer.parseInt(br.readLine());
			int mid = N / 2;
			int result = 0;
			for(int i = 0; i < N; i++) {
				String line = br.readLine();
				for(int j = 0; j < N; j++) {
					if(Math.abs(i - mid) + Math.abs(j - mid) <= mid) result += line.charAt(j) - '0';
				}
			}
			sb.append("#").append(testCase).append(" ").append(result).append("\n");
		}
		System.out.print(sb);
	}
}
